code/modules/events/meteor_wave.dm

```diff
@@ -16,6 +16,7 @@
 	startWhen		= 6
 	endWhen			= 66
 	announceWhen	= 1
+	threat = 15
```

Comment:
Considering meteor shields, unidirectional waves and on-station asteroids ruins, the threat should be slightly lower.

Reply:
This is only calculated during the meteor storm itself, mind. Basically the purpose of this threat increase is to allow the crew to focus on meteors for a bit, it doesn't represent the damage to be done as much as how much the crew needs to focus on it during.